saleor/graphql/schema.graphql

```diff
@@ -1831,11 +1831,55 @@ type Image {
   alt: String
 }
 
+type Invoice implements Job & Node & ObjectWithMetadata {
```

Comment:
I'm not sure the invoice should implement the `Job` interface? Excuse me If I'm wrong, but I thought in cases like generating files we want to first return a job that you can keep querying to check the status of the operation. Once the job is done it will return the invoice. Was it supposed to work like that @korycins @patrys?

Reply:
The idea was to make long-running tasks implement the interface so you can monitor the object itself instead of an abstract job class.

Reply:
Sounds good. We will need to make sure that we have it consistent with other jobs, e.g. CSV exports.